pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
```

Comment:
Just removed default version from api.proto (proto3 doesn't support default value) and document the version controlling in the proposal. After that, `protoc-gen-gogo` is worked for proto3 as this proposal doesn't contain `any` and `json` filed.

@yujuhong @mtaufen Should we switch to proto3, or keep it as is for consistency with other kubernetes apis?